### ISSUE TITLE
Show backlinks from `:PROPERTIES:`

### ISSUE THREAD

**Issue by zetashift** [chipsenkbeil/org-roam.nvim#41]
So I have a reading list, with books for example:

```org
* TOREAD Book Name
:PROPERTIES:
:author: [[id:734f176d-f598-460c-a2f9-c134e5d339ca][Author Name]]  
:END:
```

Now when I'm on the authors page I expect to see backlink to the that properties drawer/book name. Currently the quickfix window is completely empty. 
Is it possible to do this or is there maybe a better way to organize this?

**Comment by chipsenkbeil** [comment]
Does [orgmode](https://orgmode.org/) support links in the properties drawer? This plugin uses [nvim-orgmode](https://github.com/nvim-orgmode/orgmode) to find links within a file and then index id links, so it would need to be handled upstream in that plugin to support links there. In the meantime, you'd need to include your link somewhere else (heading or body) for it to be available.

cc @kristijanhusak

**Comment by zetashift** [comment]
So in Doom Emacs I can go to links in properties drawer, but from looking around this seems to be not-according-to-spec.

From: https://www.orgroam.com/manual.html

> Org-roam relied on the obtained Org AST for the buffer to parse links. However, links appearing in some places (e.g. within property drawers) are not considered by the Org AST to be links. Therefore, Org-roam takes special care of additionally trying to process these links. Use org-roam-db-extra-links-elements to specify which additional Org AST element types to consider.

**Comment by chipsenkbeil** [comment]
If the org AST doesn't support links in property drawers by default, then I don't believe nvim-orgmode will support them, either.

I'm totally happy to have this feature added, but can't guarantee I'll get to this myself. Happy to review a pull request, though!

**Comment by zetashift** [comment]
> If the org AST doesn't support links in property drawers by default, then I don't believe nvim-orgmode will support them, either.

I believe that's completely fair!

> I'm totally happy to have this feature added to this plugin, but can't guarantee I'll get to this myself. Happy to review a pull request, though!

Well it doesn't help that I'm new to orgmode/neovim as well :P. 
I could atleast attempt it, I'm not sure where to start, wouldn't this require a change in the tree-sitter parser to identify links in the `PROPERTIES` drawer?

**Comment by kristijanhusak** [comment]
You can see how it's done in nvim-orgmode https://github.com/nvim-orgmode/orgmode/blob/master/lua/orgmode/files/file.lua#L679-L697.

**Comment by zetashift** [comment]
Oh that is pretty cool!

That would entail something like this:

```lua
  local ts_query = ts_utils.get_query([[
      (paragraph (expr) @links)
      (drawer (contents (expr) @links))
      (property_drawer (value (value) @links))
      (headline (item (expr)) @links)
  ]])
  ```
The ` (property_drawer (value (value) @links))` got added in?

**Comment by kristijanhusak** [comment]
@zetashift yes, something like that, but you would need to write the surrounding logic also to get the links for org-roam.
